### PR TITLE
[dhctl] Prevent to break already bootstrapped cluster when bootstrap new cluster

### LINKED ISSUE #1657: [dhctl] Prevent ruining existing cluster
### Preflight Checklist

- [X] I agree to follow the [Code of Conduct](https://github.com/deckhouse/deckhouse/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/deckhouse/deckhouse/issues) for an issue that matches the one I want to file, without success.

### Use case. Why is this important?

When running `dhctl bootstrap` one can use host with existing cluster and dhctl will replace existing resources, especially cm/deckhouse, so cluster may become unusable:

screen

### Proposed Solution

`dhctl bootstrap` should check for existing resources after successful connect to KubernetesAPI. Confirmation dialog or force flag can prevent disasters.

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by name212**
Signed-off-by: Nikolay Mitrofanov <nikolay.mitrofanov@flant.com>

## Description
Prevent to break already bootstrapped cluster when bootstrap new cluster.

Before apply deckhouse manifests, we check cluster UUID now, and if we will get different uuid (none empty) from cache and from cluster, we will return error. 

## Why do we need it, and what problem does it solve?
Close #1657

## Changelog entries

```changes
section: dhctl
type: feature
summary: Prevent to break already bootstrapped cluster when bootstrap new cluster
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @name212. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @name212. I've started the workflow for label 'e2e/run/static'...

File: dhctl/pkg/kubernetes/actions/deckhouse/install.go [hunk 1]
```diff
@@ -307,11 +307,11 @@ func CreateDeckhouseManifests(kubeCl *client.KubernetesClient, cfg *Config) erro
 				return manifests.ClusterUUIDConfigMap(cfg.UUID)
 			},
 			CreateFunc: func(manifest interface{}) error {
-				_, err := kubeCl.CoreV1().ConfigMaps("kube-system").Create(context.TODO(), manifest.(*apiv1.ConfigMap), metav1.CreateOptions{})
+				_, err := kubeCl.CoreV1().ConfigMaps(manifests.ClusterUUIDCmNamespace).Create(context.TODO(), manifest.(*apiv1.ConfigMap), metav1.CreateOptions{})
 				return err
 			},
 			UpdateFunc: func(manifest interface{}) error {
-				_, err := kubeCl.CoreV1().ConfigMaps("kube-system").Update(context.TODO(), manifest.(*apiv1.ConfigMap), metav1.UpdateOptions{})
+				_, err := kubeCl.CoreV1().ConfigMaps(manifests.ClusterUUIDCmNamespace).Update(context.TODO(), manifest.(*apiv1.ConfigMap), metav1.UpdateOptions{})
```
Comment (nabokihms) [inline dhctl/pkg/kubernetes/actions/deckhouse/install.go:314]: NIT: 

I personally do not like declaring constants with long names, when they are the part of an object. As for me, it better to get the name from an object.
```go
clusterUUIDCm := manifest.(*apiv1.ConfigMap)
fmt.Println(clusterUUIDCm.Name, clusterUUIDCm.Namespace)
```
or

```go
clusterUUIDCm := manifests.ClusterUUIDConfigMap()
fmt.Println(clusterUUIDCm.Name, clusterUUIDCm.Namespace)
```

Comment (nabokihms) [reply dhctl/pkg/kubernetes/actions/deckhouse/install.go:314]: But is it not a blocker for this PR.

**Review by nabokihms — APPROVED** [review]

**Pull request merged**